oneview_redfish_toolkit/blueprints/event_service.py

```diff
@@ -37,11 +39,32 @@ def get_event_service():
         /redfish/v1/SessionService is requested.
 
         Returns:
-                JSON: JSON with EventService.
+            JSON: JSON with EventService.
+
+        Exceptions:
+            OneViewRedfishError: if there are invalid integers in
+                DeliveryRetryAttempts and DeliveryRetryIntervalSeconds
+                on the CONFIG file.
     """
     try:
+        try:
+            delivery_retry_attempts = \
+                int(util.config["event_service"]["DeliveryRetryAttempts"])
```

Comment:
So we're not using the globals anymore... right?

Reply:
I changed that. Now we're using it.